flexxamples/howtos/python_side_widget2.py

```diff
@@ -0,0 +1,49 @@
+from flexx import flx
+
```

Comment:
A short docstring here would be nice as well.

Reply:
Done.